pepys_import/core/store/data_store.py

```diff
@@ -2096,17 +2096,16 @@ def find_dependent_objects(self, table_name: str, id_list: list) -> dict:
 
         return output
 
-    def delete_objects(self, table_obj, id_list):
+    def delete_objects(self, table_name, id_list):
         """
         Deletes the given objects.
 
-        :param table_obj: A table object, or name of the table that IDs belong to
-        :type table_obj: SQLAlchemy Model or str
+        :param table_name: Name of the table that IDs belong to
+        :type table_name: str
         :param id_list: List of objects IDs
         :type id_list: list
         """
-        if isinstance(table_obj, str):
-            table_obj = self._get_table_object(table_obj)
+        table_obj = self._get_table_object(table_name)
```

Comment:
I think I might not have explained myself properly: I was hoping that both `delete_objects` and `find_dependent_objects` could take a parameter which could be _either_ a table name, or a table object. This change actually removes this from `delete_objects` - I was hoping the logic that was in `delete_objects` could be added to `find_dependent_objects` as well. Does that make sense?

Reply:
Aah, my mistake. Yes, it makes sense. I changed the code: https://github.com/debrief/pepys-import/pull/789/commits/d8544fcfa31ff15ba25727b96d39f7f0d0c8129f

Reply:
Great, thanks - I'll approve now.